atlp-rwanda/rca-barefoot-team6-fe
# feat: login and register done

#### What does this PR do?
 
Implementing login and register on frontend.

#### Description of  the task to be completed?

I have done implementation of login and register on frontend part with user

#### How should this be manually tested?

Check out this repo and then run `npm i` after successful installation run `npm run dev` then go into browser at this URL `{BASE_URL}/auth/login` or `{BASE}/auth/register` for register.

#### Any background context you want to provide?

I have used `react-query` so someone implementing other request via frontend should understand it.

#### What are the relevant pivotal tracker stories?



#### Screenshots (if appropriate)

<img width="443" alt="image" src="https://user-images.githubusercontent.com/73703812/232459632-d8f5f831-b103-4fd5-b418-18eb0415b948.png">

<img width="405" alt="image" src="https://user-images.githubusercontent.com/73703812/232459702-e3f4e1c0-8ccd-407e-9d61-161b1e290c83.png">
